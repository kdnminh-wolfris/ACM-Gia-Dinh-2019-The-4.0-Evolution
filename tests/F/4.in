8308 7201 5574 1264