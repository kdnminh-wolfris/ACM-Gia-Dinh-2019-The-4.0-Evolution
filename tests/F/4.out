7044 8308